src/sx-jest-snapshot-serializer/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import { __internal as __sxInternal } from '@adeira/sx';
+import prettier from 'prettier';
+import prettyFormat from 'pretty-format';
+
+/* global window */
+
+expect.addSnapshotSerializer({
+  test: (val) => {
+    return (
+      val &&
+      (val.$$typeof === Symbol.for('react.test.json') || // react-test-renderer
+        val instanceof window.HTMLElement) // @testing-library/react
+    );
+  },
+  print: (val) => {
+    const printedStyles = prettier.format(
+      __sxInternal.StyleCollector.print(), // TODO: apply `stylis` rules to match what SX actually does
+      { filepath: 'test.css' },
```

Comment:
I find setting a parser more readable instead of the `filepath`
```suggestion
      { parser: 'css' },
```

Reply:
I also always found the `filepath` awkward. Thanks for showing me this option - I didn't know it. 🙂